templates/cisco_ios_show_environment_temperature.template

```diff
@@ -0,0 +1,41 @@
+Value SWITCH (\d+)
+Value SWITCH_TEMPERATURE (\w+)
+Value INLET_TEMPERATURE_VALUE (\d+)
+Value INLET_TEMPERATURE_STATE (\w+)
+Value INLET_YELLOW_THRESHOLD (\d+)
+Value INLET_RED_THRESHOLD (\d+)
+Value HOTSPOT_TEMPERATURE_VALUE (\d+)
+Value HOTSPOT_TEMPERATURE_STATE (\w+)
+Value HOTSPOT_YELLOW_THRESHOLD (\d+)
+Value HOTSPOT_RED_THRESHOLD (\d+)
+
+Start
+  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
```

Comment:
```suggestion
  ^Switch\s+\d -> Continue.Record
  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
```

Reply:
Made changes.  Works.  Moving to next suggestion.